app/src/pages/organization/organizationProjectsPage/projectsPageHeader/projectsPageHeader.jsx

```diff
@@ -77,7 +94,25 @@ export const ProjectsPageHeader = ({ hasPermission, onCreateProject }) => {
         <div className={cx('actions')}>
           {isNotEmpty && (
             <div className={cx('icons')}>
-              <i className={cx('search-icon')}>{Parser(searchIcon)}</i>
+              <SearchEntitiesURLContainer
```

Comment:
Can we use `withFilter` HOC instead from `src/controllers/filter/withFilter.jsx` as it is done for FiltersPage? (src/pages/inside/filtersPage/filtersPage.jsx).